e2e/e2e.go

```diff
@@ -76,18 +75,26 @@ func BuildDependenciesList(ctx context.Context) ([]string, error) {
 
 // findDependenciesInRepo finds the dependencies in a repository
 // at all of the possible paths
-func findDependenciesInRepo(serviceName, repoPath string) map[string]struct{} {
+func findDependenciesInRepo(ctx context.Context, serviceName string) (map[string]struct{}, error) {
 	possibleFiles := []string{"devenv.yaml", "noncompat-service.yaml", "service.yaml"}
+	gh, err := githubauth.NewClient()
+	if err != nil {
+		return nil, err
+	}
 
 	var dc *DevenvConfig
 	for _, f := range possibleFiles {
-		if _, err := os.Stat(filepath.Join(repoPath, f)); err != nil {
+		config, _, _, err := gh.Repositories.GetContents(ctx, "getoutreach", serviceName, f, nil)
```

Comment:
I think this would apply here as well: https://github.com/getoutreach/devenv/pull/494#discussion_r1129633388